packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -0,0 +1,310 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { color } from '../../utils/colors';
+
+const rangeHeight = () => css( { height: 30, minHeight: 30 } );
+const rootRtl = ( { isRTL } ) => css( { direction: isRTL ? 'rtl' : 'ltr' } );
+
+export const Root = styled.span`
+	-webkit-tap-highlight-color: transparent;
+	box-sizing: border-box;
+	cursor: pointer;
+	align-items: flex-start;
+	display: inline-flex;
+	justify-content: flex-start;
+	padding: 0;
+	position: relative;
+	touch-action: none;
+	width: 100%;
+
+	${rootRtl}
+`;
+
+const wrapperMargin = ( { marks } ) =>
+	css( { marginBottom: marks ? 16 : null } );
+
+export const Wrapper = styled.span`
+	box-sizing: border-box;
+	color: ${color( 'blue.medium.focus' )};
+	display: block;
+	padding-top: 15px;
+	position: relative;
+	margin-left: 10px;
+	width: 100%;
+
+	${rangeHeight};
+	${wrapperMargin};
+
+	[dir='rtl'] & {
+		margin-right: 10px;
+	}
+`;
+
+export const BeforeIconWrapper = styled.span`
+	margin-right: 6px;
+	margin-top: 5px;
+
+	> .dashicon {
```

Comment:
Where are we using Dashicons here? can we replace them with an icon from the icons package?

Reply:
@youknowriad I preserved the same functionality as the previous version, which uses Dashicon.

I just pushed an update that removes `Dashicon` usage and opts for the new `@wordpress/icon` instead. (Tests have also been updated)